src/validations/systemIntakeSchema.ts

```diff
@@ -163,6 +164,34 @@ export const DateValidationSchema: any = Yup.object().shape(
       },
       then: Yup.string().required('The year is required')
     }),
+    validGrtDate: Yup.string().when(
+      ['grtDateDay', 'grtDateMonth', 'grtDateYear'],
+      {
+        is: (grtDateDay: string, grtDateMonth: string, grtDateYear: string) => {
+          // Only check for a valid date if monday, date, and year a filled
+          if (grtDateDay && grtDateMonth && grtDateYear) {
+            // If the date is valid, it passes the validation
+            if (
+              DateTime.fromObject({
```

Comment:
This looks totally fine, and is probably the way I would have tackled it. Just out of curiosity, did you look in yups `date` function? `yup.date()`

Reply:
Yeah, I didn't find much on it. It looked like it only supported date ranges.

Reply:
And I think to do it, we'd need to pass a Javascript date to it.

I was playing around and this is valid, so I think we'd have issues:
```
new Date(2020, 0, 45)
// Fri Feb 14 2020 00:00:00 GMT-0800 (Pacific Standard Time)
```